terms/lossfunctions_ag.bib

```diff
@@ -0,0 +1,56 @@
+@InProceedings{pmlr-v32-yanga14,
+  title = 	 {The Coherent Loss Function for Classification},
+  author = 	 {Wenzhuo Yang and Melvyn Sim and Huan Xu},
+  booktitle = 	 {Proceedings of the 31st International Conference on Machine Learning},
+  pages = 	 {37--45},
+  year = 	 {2014},
+  editor = 	 {Eric P. Xing and Tony Jebara},
+  volume = 	 {32},
+  number =       {1},
+  series = 	 {Proceedings of Machine Learning Research},
+  address = 	 {Bejing, China},
+  month = 	 {22--24 Jun},
+  publisher = 	 {PMLR},
+  pdf = 	 {http://proceedings.mlr.press/v32/yanga14.pdf},
```

Comment:
There are many useless entries here ... please remove.  See rule 8. in the guidelines.